### PR TITLE
[DEV-3122] Standard portal mint templates + components

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Includes default templates for partner portals, new components, new stories added, some demo hooks added/improved.

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: https://saasquatch.atlassian.net/browse/DEV-3122
 - Process.st launch checklist: https://app.process.st/runs/Mint-Components-122-hofvo7IhFol_wCuyE5xOBg
 - Stencilbook: https://mint-components.stencilbook.saasquat.ch/pr198/index.html

### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run
- [x] The behaviour changes in the pull request are covered by specs
- [x] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist
- [x] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/mint-components/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.2] - 2021-09-27
+
+### Changed
+
```
Comment (DerekSiemens) [inline packages/mint-components/CHANGELOG.md:13]: Missing Hero unit and missing change to registration component

File: packages/mint-components/package.json [hunk 2]
```diff
@@ -1,10 +1,7 @@
 {
   "name": "@saasquatch/mint-components",
-  "version": "1.2.1",
```
Comment (DerekSiemens) [inline packages/mint-components/package.json:3]: Why were these deleted?

Comment (00salmon) [reply packages/mint-components/package.json:3]: no idea, they have returned!

**Review by DerekSiemens — APPROVED** [review]
